Stream_HomeWork_10/src/Main.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+
+	public static void main(String[] args) {
+		Person p1 = new Person("Ivan",10);
+		Person p2 = new Person("Ian",11);
+		Person p3 = new Person("Anton",19);
+		Person p4 = new Person("Sergey",22);
+		Person p5 = new Person("Andrew",14);
+		Person p6 = new Person("Denis",18);
+		List<Person> input = Arrays.asList(p1,p2,p3,p4,p5,p6);
+		solution1(input);
+	}
+	public static String solution(List<Person> input){
```

Comment:
It is better to have better names since just 'solution' is not clear for which task the solution is referring to.